base/applications/regedit/lang/bg-BG.rc

```diff
@@ -643,7 +643,7 @@ END
 /* Dialogs */
 
 IDD_EXPORTRANGE DIALOGEX 50, 50, 370, 50
-STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS | WS_BORDER
+STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_CLIPSIBLINGS | WS_BORDER
```

Comment:
Please check that the Save-As dialog (when e.g. exporting a registry key) still correctly shows the extra thing.

Reply:
What do you mean with "the extra thing"?

Reply:
![export_key](https://github.com/user-attachments/assets/2962eefe-1ce5-4ca0-b6d1-fad2d00a6aaf)
This :point_up:

Reply:
To me the export dlg looks good:
![image](https://github.com/user-attachments/assets/911b7cbd-c747-4495-b666-e196b2e66a13)
registry-export-dlg is modal.

Reply:
running our patched regedit on Windows XP SP3, does look also good:
![image](https://github.com/user-attachments/assets/8b9a69c8-c67a-4e45-925c-e2687b4c221d)

Reply:
should be good then :+1: